api/docgen/docgen.go

```diff
@@ -406,6 +406,42 @@ func init() {
 	percent := types.Percent(123)
 	addExample(percent)
 	addExample(&percent)
+
+	addExample(&types.ActorEventBlock{
+		Codec: 0x51,
+		Value: []byte("data"),
+	})
+
+	addExample(&types.ActorEventFilter{
+		Addresses: []address.Address{addr},
+		Fields: map[string][]types.ActorEventBlock{
+			"abc": {
+				{
+					Codec: 0x51,
+					Value: []byte("data"),
```

Comment:
it just so happens that if you stick an extra `d` in front of `data` that you turn it into valid cbor for the string `"data"`, maybe making it valid would be a bonus here for docs?

Reply:
FYI:

```
$ cborg bin2diag ddata
64                                                # string(4)
  64617461                                        #   "data"
```

Reply:
Done.